src/coredump/stacktrace.c

```diff
@@ -206,14 +205,9 @@ static int parse_package_metadata(const char *name, JsonVariant *id_json, Elf *e
                                 }
 
                                 /* Finally stash the name, so we avoid double visits. */
-                                name_key = strdup(name);
-                                if (!name_key) {
-                                        log_oom();
-                                        return DWARF_CB_ABORT;
-                                }
-                                r = set_ensure_consume(c->modules, &string_hash_ops, name_key);
+                                r = set_put_strdup(c->modules, name);
```

Comment:
hmm, set_put_strdup_full(), no? (did i suggest the one without?)

Reply:
isn't it the same? I do want string_hash_ops, no?

`#define set_put_strdup(s, p) set_put_strdup_full(s, &string_hash_ops_free, p)`

Reply:
oh ah